src/Okta.Idx.Sdk/EnrollOktaVerifyAuthenticatorOptions.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Okta.Idx.Sdk
+{
+    public class EnrollOktaVerifyAuthenticatorOptions : SelectEnrollAuthenticatorOptions
+    {
+        public string Channel { get; set; }
```

Comment:
I wonder if this should be an AuthenticatorMethodType instead of string? Thoughts?